libraries/lib-module-manager/AsyncPluginValidator.h

```diff
@@ -0,0 +1,64 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file AsyncPluginValidator.h
+
+  @author Vitaly Sverchinsky
+
+  Part of lib-module-manager library
+
+**********************************************************************/
+
+#pragma once
+
+#include <wx/string.h>
+#include <memory>
+
+class PluginDescriptor;
+
+
+/**
+ * \brief Used to perform plugin validation in a dedicated process.
```

Comment:
Maybe less misleading to write, "Starts and communicates with a dedicated process to perform plugin validation".

But the member functions of it are available to call in the main process.